docs/projects.md

```diff
@@ -183,3 +183,4 @@
 **Project**|**Sponsor**|**TOC Deck**|**Archived**|**Maturity Level**
 :-----:|:-----:|:-----:|:-----:|:-----:
 [rkt](http://rkt.io)|Brian Grant|[3/15/17](https://docs.google.com/presentation/d/1KzA58_Zz30mKKzeLuSvXLh63aIC75KRdAOTw4PJ_10g/edit?usp=sharing)|[8/16/19](https://www.cncf.io/blog/2019/08/16/cncf-archives-the-rkt-project/)|Archived
+[OpenTracing](http://opentracing.io/)|Bryan Cantrill|[8/17/16](https://docs.google.com/presentation/d/1kQkmJtT0bjSRvUTP5YFTKaXSfIM3aL7zxja_KtZtbgw/edit#slide=id.g15fc45ec1a_0_165)|[10/11/16](https://cncf.io/news/blogs/2016/10/opentracing-joins-cloud-native-computing-foundation)|Archived
```

Comment:
The **Archived** date is currently the accepted date (and link). 
This should probably be Aug. 26 with a link to https://www.cncf.io/blog/2021/08/26/opentelemetry-becomes-a-cncf-incubating-project/?